core/src/main/java/com/linecorp/armeria/common/NonWrappingRequestContext.java

```diff
@@ -191,19 +193,35 @@ public final MeterRegistry meterRegistry() {
         return meterRegistry;
     }
 
+    /**
+     * Returns {@link AttributeMap}.
+     */
+    public AttributeMap attributeMap() {
+        return attributeMap;
```

Comment:
Not sure why we need this method. Doesn't this class already implement `AttributeMap` and delegate to `attributeMap`?